modules/nw-metallb-software-components.adoc

```diff
@@ -5,31 +5,27 @@
 [id="nw-metallb-software-components_{context}"]
 = MetalLB software components
 
-When you install the MetalLB Operator, the `metallb-operator-controller-manager` deployment starts a pod.
-The pod is the implementation of the Operator.
-The pod monitors for changes to all the relevant resources.
+When you install the MetalLB Operator, the `metallb-operator-controller-manager` deployment starts a pod. The pod is the implementation of the Operator. The pod monitors for changes to all the relevant resources.
 
 When the Operator starts an instance of MetalLB, it starts a `controller` deployment and a `speaker` daemon set.
 
 `controller`::
-The Operator starts the deployment and a single pod.
-When you add a service of type `LoadBalancer`, Kubernetes uses the `controller` to allocate an IP address from an address pool.
+The Operator starts the deployment and a single pod. When you add a service of type `LoadBalancer`, Kubernetes uses the `controller` to allocate an IP address from an address pool.
+In case of a service failure, ensure you have the following entry in your `controller` pod logs:
++
+[source,terminal]
```

Comment:
```suggestion
In case of a service failure, verify you have the following entry in your `controller` pod logs:
+
.Example output
[source,terminal]
```
Just a small nit. See the [repo guidelines ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) for adding an label to an example output. Also, _ensure_ is not wrong, but I think in our docs it often implies, "make sure you did this thing." To me, _verify_ is a bit clearer in this case. WDYT?

Reply:
Agreed. Verify sounds more better

Reply:
addressed

Reply:
Addressed